service/src/main/java/org/kiwiproject/champagne/resource/HostConfigurationResource.java

```diff
@@ -39,12 +42,14 @@ public class HostConfigurationResource extends AuditableResource {
 
     private final HostDao hostDao;
     private final ComponentDao componentDao;
+    private final TagDao tagDao;
 
-    public HostConfigurationResource(HostDao hostDao, ComponentDao componentDao, AuditRecordDao auditRecordDao, ApplicationErrorDao errorDao) {
+    public HostConfigurationResource(HostDao hostDao, ComponentDao componentDao, TagDao tagDao, AuditRecordDao auditRecordDao, ApplicationErrorDao errorDao) {
```

Comment:
Getting to a point where it might make sense to create a "Context" object to pass around?

Reply:
Yeah maybe